syntax = "proto3";
package resource_monitor_config;





/*
ListResourceMonitorConfig请求
*/
message ListResourceMonitorConfigRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 pageSize = 2 ;
}

/*
ListResourceMonitorConfig返回
*/
message ListResourceMonitorConfigResponse {
message List {
    /*
    实例ID
    */
    string id = 1 ;
    /*
    enabled
    */
    bool enabled = 2 ;
    /*
    资源ID
    */
    string objectId = 3 ;
    /*
    资源名称
    */
    string objectName = 4 ;
}
        
    
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated List list = 4 ;
}

/*
ListResourceMonitorConfigApi返回
*/
message ListResourceMonitorConfigResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    resource_monitor_config.ListResourceMonitorConfigResponse data = 4 ;
}

